server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionCoordinator.java

```diff
@@ -313,6 +291,61 @@ public void run() {
     LOG.info("Shutting down");
   }
 
+  private void updateSummaries() {
+    ExecutorService executor =
+        ThreadPools.createFixedThreadPool(10, "Compaction Summary Gatherer", false);
```

Comment:
Should the number of threads (10 here) be some factor (like 10%) of the # of tservers in tServerSet?

Reply:
I Was thinking of % up to some max (was thinking a max of 10)... and then just decided on the max... that should get through even a large number of tsevers in short order, except when there are a good bit of unresponsive tservers... was also wondering if it should be configurable, but was thinking 10 should be good and we can make it configurable if ever needed